resources/views/oauth2/consent.blade.php

```diff
@@ -1,50 +1,50 @@
-                </div>
-            </div>
-            <legend>This app would like to:</legend>
-            <div class="consents-container">
-            <ul class="list-unstyled">
-            @foreach ($requested_scopes as $scope)
-                <li> {!!$scope->getShortDescription()!!}&nbsp;<span class="glyphicon glyphicon-info-sign pointable info" aria-hidden="true" data-content="{!! $scope->getDescription() !!}" title="Scope Info"></span></li>
-            @endforeach
-            </ul>
-            </div>
-            <div class="consents-footer">
-            <p class="privacy-policy">** <b>{!!$app_name!!}</b> Application and <b>{!! Config::get('app.tenant_name') !!}</b> will use this information in accordance with their respective <a target='_blank' href="{!!$tos_uri!!}">terms of service</a> and <a target='_blank' href="{!!$policy_uri!!}">privacy policies</a>.</p>
-            {!! Form::open(array('url' => URL::action("UserController@postConsent") ,'id'=>'authorization_form', 'method' => 'post',  "autocomplete" => "off", "target"=> "_self" )) !!}
-                <input type="hidden"  name='trust' id='trust' value=""/>
-                <button class="btn btn-default btn-md btn-consent-action" id="cancel-authorization" type="button">Cancel</button>
-                <button class="btn btn-primary btn-md btn-consent-action" id="approve-authorization" type="button">Accept</button>
-            {!! Form::close() !!}
-            </div>
-        </div>
-    </div>
-</div>
-@stop
+@section('scripts')
+    <script>
+
+        let scopes = [];
+
+        @foreach($requested_scopes as $scope)
+        scopes.push('{!! $scope->getShortDescription() !!}');
+        @endforeach
+
+        const disclaimer = '<p class="privacy-policy">** <b>{!!$app_name!!}</b> Application and <b>{!! Config::get('app.tenant_name') !!}</b> will use this information in accordance with their respective <a target="_blank" href="{!!$tos_uri!!}">terms of service</a> and <a target="_blank" href="{!!$policy_uri!!}">privacy policies</a>.</p>';
+
+        let config = {
+            requestedScopes: scopes,
+            csrfToken: document.head.querySelector('meta[name="csrf-token"]').content,
+            userName: '{{ Session::has('username') ? Session::get('username') : ""}}',
+            realm: '{{isset($identity_select) ? $realm : ""}}',
+            appName: '{{ Config::get("app.app_name") }}',
+            appDescription: '{{ str_replace(array("\n", "\r"), '', strip_tags(Config::get("app.app_info"))) }}',
+            appLogo: '{{  Config::get("app.logo_url") }}',
```

Comment:
app logo its the oauth2 app logo

Reply:
Changed to 

- appLogo: '{!! $app_logo !!}' for the client app logo
- idpLogo: '{{  Config::get("app.logo_url") }}' for the IDP app logo